packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -131,137 +74,177 @@ governing permissions and limitations under the License.
 
-  .spectrum-Steplist-markerContainer {
-    left: auto;
-    right: 0;
+/* Large size */
+.spectrum-Steplist--large {
+  line-height: var(--spectrum-steplist-marker-diameter-large);
 
-    margin-left: 0;
-    margin-right: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+  .spectrum-Steplist-label {
+    font-size: var(--spectrum-global-dimension-font-size-150);
   }
-
-  .spectrum-Steplist-segment {
-    right: var(--spectrum-steplist-marker-diameter);
-    left: auto;
+  .spectrum-Steplist-chevron {
+    transform: scale(1.25);
   }
-}
 
-.spectrum-Steplist-item:first-child,
-.spectrum-Steplist-item:last-child {
-  width: calc(calc(var(--spectrum-steplist-step-width) / 2) - calc(var(--spectrum-steplist-marker-diameter) * 2.5));
+  .spectrum-Steplist-item {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-large) * 1.5) auto;
+    margin-left: calc(var(--spectrum-steplist-marker-diameter-large) / 2);
+  }
+  .spectrum-Steplist-item > a {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-large) * 2) auto;
+    
+    .spectrum-Steplist-marker {
+      line-height: var(--spectrum-steplist-marker-diameter-large);
+      width: var(--spectrum-steplist-marker-diameter-large);
+      height: var(--spectrum-steplist-marker-diameter-large);
+      border-radius: var(--spectrum-steplist-marker-diameter-large);
+      font-size: var(--spectrum-global-dimension-font-size-200);
+    }
+  }
 }
 
-/* Single step */
-.spectrum-Steplist-item:only-child {
+/* Large size */
+.spectrum-Steplist--xlarge {
+  line-height: var(--spectrum-steplist-marker-diameter-xlarge);
+  
   .spectrum-Steplist-label {
-    /* Be centered */
-    left: 50%;
-
-    /* Override last-child bits */
-    transform: translate(-50%);
+    font-size: var(--spectrum-global-dimension-font-size-300);
+  }
+  
+  .spectrum-Steplist-chevron {
+    transform: scale(1.5);
   }
 
-  .spectrum-Steplist-markerContainer {
-    /* Be centered */
-    left: 50%;
-
-    /* Override last-child bits */
-    margin-right: 0;
-    margin-left: calc(calc(var(--spectrum-steplist-marker-hitArea) / 2) * -1);
+  .spectrum-Steplist-item {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-xlarge) * 1.5) auto;
+    margin-left: calc(var(--spectrum-steplist-marker-diameter-xlarge) / 2);
+  }
+  .spectrum-Steplist-item > a {
+    grid-template-columns: calc(var(--spectrum-steplist-marker-diameter-xlarge) * 2) auto;
+    
+    .spectrum-Steplist-marker {
+      line-height: var(--spectrum-steplist-marker-diameter-xlarge);
+      width: var(--spectrum-steplist-marker-diameter-xlarge);
+      height: var(--spectrum-steplist-marker-diameter-xlarge);
+      border-radius: var(--spectrum-steplist-marker-diameter-xlarge);
+      font-size: var(--spectrum-global-dimension-font-size-400);
+    }
   }
 }
+
+/* First step overrides */
+.spectrum-Steplist-item:first-child {
+  grid-template-columns: auto;
+  grid-template-areas: 
+    "link";
```

Comment:
grid areas with auto should just collapse automagically, you shouldn't need to override the template. did you run into an issue with it?

Reply:
gaps should collapse as well